lib/cli/options.ts

```diff
@@ -42,11 +41,11 @@ export class Option {
     if (value && typeof value === 'string') {
       return '' + value;
     }
-    return null;
+    return '';
```

Comment:
`null` is a valid string in typescript and `android-platforms` as a custom default value.